waku/waku_test.go

```diff
@@ -1598,8 +1650,12 @@ func TestRateLimiterIntegration(t *testing.T) {
 	p := p2p.NewPeer(enode.ID{1}, "1", []p2p.Cap{{"waku", 0}})
 	rw1, rw2 := p2p.MsgPipe()
 	defer func() {
-		rw1.Close()
-		rw2.Close()
+		if err := rw1.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
+		if err := rw2.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)